mailet/base/src/test/java/org/apache/mailet/base/test/FakeMailContext.java

```diff
@@ -44,11 +49,17 @@
         private final MailAddress sender;
         private final Collection<MailAddress> recipients;
         private final MimeMessage msg;
+        private final List<String> attributeNames;
```

Comment:
No it is not. This attributes are the one of a mail.
The one belows are more like configuration properties for a mailet.
Sorry, I did not choose the domain vocabulary....